src/frontend/efiling-frontend/src/components/page/package-confirmation/PackageConfirmation.js

```diff
@@ -26,9 +26,29 @@ const uploadButton = {
 };
 
 // TODO: Fix values
-const summaryTableData = [
-  { name: "Statutory Fees:", value: "$100.00", isValueBold: true },
-  { name: "Number of Documents in Package:", value: "10", isValueBold: true }
+const feesData = [
+  {
+    name: "Statutory Fees:",
+    value: "$100.00",
+    isValueBold: true
+  },
+  {
+    name: "Number of Documents in Package:",
+    value: "10",
```

Comment:
Is this not more like test data that should be in storybook or retrieved from the API rather than hardcoded?

Reply:
yeah when I know how this data is coming in, then was going to change
just for time being I didnt wanna spend time refactoring cause itll go away / change a lot when we actually have the backend code